### ISSUE TITLE
Problem with Height

### ISSUE THREAD

**Issue by davidchristian1** [hadukin/custom_sliding_segmented_control#29]
I tried to modify the height of segmented control. The content's height shrinks, but the container and thumb container not following the content height. Here is the code:
```dart
CustomSlidingSegmentedControl(
                innerPadding: EdgeInsets.zero,
                isStretch: true,
                height: 20,
                padding: 0,
                initialValue: filter,
                thumbDecoration: BoxDecoration(color: AppColor.primaryColor, borderRadius: BorderRadius.circular(50)),
                decoration:  BoxDecoration(color: AppColor.segmentedControlColor, borderRadius: BorderRadius.circular(50)),
                onValueChanged: (Filter? value) {
                  if (value != null) {
                    setState(() {
                      filter = value;
                    });
                  }
                },
                children: <Filter, Widget> {
                  Filter.a: Text(
                    'A',
                  ),
                  Filter.b: Text(
                    'B',
                  ),
                  Filter.c: Text(
                    'C',
                  )
    }
)
```
And here is the result:
<img width="377" alt="Screenshot 2023-05-25 at 10 45 32" src="https://github.com/hadukin/custom_sliding_segmented_control/assets/47098784/02c509f6-b4ca-431e-9bcf-6b5434ccb4c6">

I already tried to reduce radius, but still getting the same result.

**Comment by hadukin** [comment]
@davidchristian1 hello, maybe you have some specific height set for the Text widget in your project

try this, most likely there is a problem in the text

```
Filter.a: Text(
    'A',
    style: TextStyle(height: 0.0),
),
Filter.b: Text(
    'B',
    style: TextStyle(height: 0.0),
),
Filter.c: Text(
    'C',
    style: TextStyle(height: 0.0),
)
```

**Comment by davidchristian1** [comment]
@hadukin Hi! I tried your code and it still doesn't work. However, your suggestion give me some idea and it is now fixed. 

The problem was I wrapped the CustomSlidingSegmentedControl with a Container that ONLY have padding and child parameters. These container somehow messed with the segmented control height, thus this issue occur. 

The solution is just change the Container to Padding and the problem is fixed.

Thank you for your time and response 🙂.
